jobs/frontend/senior-frontend-developer.md

```diff
@@ -0,0 +1,58 @@
+# Senior Frontend Developer
+
+We're looking for a pro-active senior-level frontend developer, who not only solves tasks but suggests improvements on the codebase level as well as on product and company level, and who shares knowledge within the team on meetups and in chats. <b> We are open to a person with ambitions to fill in the Team Lead role. </b>
+
+## About the job
+
+- Salary: up to 5000USD, gross per month.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually, our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- For development we usually use Kanban or similar process in Trello, Pivotal or Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
```

Comment:
[Optional]

```suggestion
- For development, we usually use Kanban or similar processes, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
```

Project management tools vary a lot by a project (e.g. on Astrohaus we use Shortcut) so I think we can omit it here.